modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js

```diff
@@ -31,7 +31,7 @@ const ResultsBar = ({
 					<span className="component-text text-truncate-inline">
 						<span className="text-truncate">
 							{Liferay.Util.sub(
-								Liferay.Language.get('x-results-for-x'),
+								(itemsTotal === 1) ? Liferay.Language.get('x-result-for-x') : Liferay.Language.get('x-results-for-x'),
```

Comment:
@sontruongces 
Can we try to use Liferay.Language.get((itemsTotal === 1) ?'x-result-for-x':'x-results-for-x')